src/main/java/org/folio/search/cql/EffectiveShelvingOrderTermProcessor.java

```diff
@@ -16,21 +11,12 @@
 @Component
 public class EffectiveShelvingOrderTermProcessor implements SearchTermProcessor {
 
-  private static final Pattern DEWEY_NUMBER_PATTERN = Pattern.compile("\\d{4}(\\.\\d+)?(\\s.{0,10}){0,10}");
-  private static final Pattern SHELF_KEY_PATTERN =
-    Pattern.compile("([A-Z]+)\\s(\\d{2,}(\\.\\d+)?)(\\s[A-Z]\\d{1,10}){0,2}(\\s.{0,10}){0,10}");
-
-  private static final List<Pattern> PATTERNS = List.of(SHELF_KEY_PATTERN, DEWEY_NUMBER_PATTERN);
-
   @Override
   public String getSearchTerm(String inputTerm) {
-    if (StringUtils.isEmpty(inputTerm) || anyMatch(PATTERNS, pattern -> pattern.matcher(inputTerm).matches())) {
-      return toRootUpperCase(inputTerm);
-    }
-
     return getValidShelfKey(new LCCallNumber(inputTerm))
       .or(() -> getValidShelfKey(new DeweyCallNumber(inputTerm)))
-      .orElse(normalizeValue(inputTerm));
+      .orElse(normalizeValue(inputTerm))
+      .trim();
```

Comment:
The same trimming as in mod-inventory-storage
![image](https://user-images.githubusercontent.com/88006020/184254432-16a3490c-74a4-4d15-bf3d-1f9c1a41c4f9.png)